tests/src/ondevice_tests/pipeline/node/segmentation_parser_test.cpp

```
@@ -379,3 +383,79 @@ TEST_CASE("SegmentationParser runtime config update changes thresholding") {
     checkMaskMatches(*outputs[0], expectedInitial, initialWidth, initialHeight);
     checkMaskMatches(*outputs[1], expectedUpdated, updatedWidth, updatedHeight);
 }
+
+TEST_CASE("SegmentationParser can set a specific head") {
+    auto description = dai::NNModelDescription{"yolo-p", "RVC4"};
+    auto archivePath = dai::getModelFromZoo(description);
+    dai::NNArchive nnArchive{archivePath};
+
+    const auto& archiveConfig = nnArchive.getConfig<dai::nn_archive::v1::Config>();
+    const auto headCount = archiveConfig.model.heads ? archiveConfig.model.heads->size() : 0;
```

Cppcheck (2.21.0)

[error] There is an unknown macro here somewhere. Configuration is required. If DEPTHAI_NLOHMANN_DEFINE_TYPE_INTRUSIVE is a macro then please configure it.

(unknownMacro)